tools/docker/docker-compose.yaml

```diff
@@ -40,6 +47,7 @@ services:
     build:
       context: ../../
       dockerfile: core/chainlink.Dockerfile
+    # entrypoint: /bin/sh -c "chainlink node import /run/secrets/keystore && chainlink node start -d -p /run/secrets/node_password -a /run/secrets/apicredentials"
```

Comment:
commented?

Reply:
yah - to be used if you want to run the tests locally